src/code/z_parameter.c

```diff
@@ -73,125 +73,21 @@ Input sPostmanTimerInput[4];
 #define RESTRICTIONS_MOON RESTRICTIONS_SET(0, 0, 0, 0, 0, 3, 3, 3, 0, 0, 0, 0)
 #define RESTRICTIONS_NO_DOUBLE_TIME RESTRICTIONS_SET(0, 0, 0, 0, 0, 3, 0, 0, 0, 0, 0, 0)
 
+#define DEFINE_SCENE(_name, enumValue, _textId, _drawConfig, restrictionFlags, _persistentCycleFlags) \
+    { enumValue, restrictionFlags },
+
+#define DEFINE_SCENE_UNSET(enumValue) { enumValue, RESTRICTIONS_NONE },
+
 RestrictionFlags sRestrictionFlags[] = {
```

Comment:
I guess this shouldn't have `SCENE_MAX` because of `RESTRICTIONS_TABLE_END` that "should" be in the table?

Reply:
This is annoying, I guess we can just not use `SCENE_MAX` here